topics/metabolomics/tutorials/lcms-preprocessing/tutorial.md

```diff
@@ -410,13 +415,13 @@ Note that you can also check the chromatograms at any moment during the workflow
 >
 > > ### {% icon tip %} Comment to W4M users
 > >
-> > In the [GTN_LCMSpreprocessing](https://galaxy.workflow4metabolomics.org/u/mpetera/h/gtnlcmspreprocessing), this step corresponds to
-the datasets number 29 and 30.
+> > In the [GTN_LCMSpreprocessing](https://workflow4metabolomics.usegalaxy.fr/u/m.petera/h/gtnlcmspreprocessing) history,
+this step corresponds to the datasets number 29 and 30.
 > {: .tip}
 >
 {: .hands_on}
 
-This tool generates Base Peak Intensity Chromatograms (BPIs) and Total Ion Chromatograms (TICs). If you provided groups like we do here, you obtain two plots:
+This tool generates Base Peak Intensity Chromatograms (BPIs) and Total Ion Chromatograms (TICs). If you provide groups as we do here, you obtain two plots:
```

Comment:
it would be nice to have a few sentences on why TIC and BPI look good. What to look for in the graphs? How would, e.g. unexpected chromatographic peaks or huge overall intensity look like?

Reply:
Again a relevant suggestion 😉 Thank you.
It is not easy to provide guidance in this due to the huge variability you get from one study to another (different devices, different designs...). Still, I added some text in 8a02dcc